dev/com.ibm.ws.junit.extensions/bnd.bnd

```diff
@@ -0,0 +1,25 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+
+bVersion=1.0
+
+instrument.disabled: true
```

Comment:
if you set `test.project: true` it won't be necessary to explicitly disable instrumentation

Reply:
done